### ISSUE TITLE
Localization for the toNaturalFileSize?

### ISSUE THREAD

**Issue by timojuonoja** [pqina/filepond#763]
**Is your feature request related to a problem? Please describe.**
If I looked correctly, the label showing the filesize below the filename is hardcoded with strings `bytes`, `KB`, `MB` and `GB`. 

**Describe the solution you'd like**
Is it possible to localize these too, because at least on our native language these are incorrect?

**Additional context**
See the attachment.
![image](https://user-images.githubusercontent.com/7699788/135015392-0ec41c25-f830-4a62-8c2b-59cf49d18579.png)

**Comment by rikschennink** [comment]
We require the issue template to be filled out on all new issues. The issue template helps us collect all the information we need to address your submission efficiently. 

The more information we have, the faster the issue can be resolved.

We will be able to more closely look at your issue once the issue template has been filled out. The issue will be closed until the above is addressed. 

**Please leave a comment after you've updated the issue.**

**Comment by timojuonoja** [comment]
I have updated the issue. Ok now?

**Comment by rikschennink** [comment]
Information missing:
- what you'd translate them to
- the name of the language

**Comment by timojuonoja** [comment]
As far as I can see..
- add to every locale file `labelFileSizeBytes`, `labelFileSizeKB`, `labelFileSizeMB` and `labelFileSizeGB`, but do you mean that I need to know every translated text for every language?
- finnish, I have already translated that by myself (see below, which also has my suggestions for finnish filesizes)

`const fi = {
  labelIdle: 'Vedä ja pudota tiedostosi tai <span class="filepond--label-action"> Selaa </span>',
  labelInvalidField: 'Kenttä sisältää virheellisiä tiedostoja',
  labelFileWaitingForSize: 'Odottaa kokoa',
  labelFileSizeNotAvailable: 'Koko ei saatavilla',
  labelFileLoading: 'Ladataan',
  labelFileLoadError: 'Virhe latauksen aikana',
  labelFileProcessing: 'Lähetetään',
  labelFileProcessingComplete: 'Lataus valmis',
  labelFileProcessingAborted: 'Lataus peruutettu',
  labelFileProcessingError: 'Virhe latauksen aikana',
  labelFileProcessingRevertError: 'Virhe palautuksen aikana',
  labelFileRemoveError: 'Virhe poistamisen aikana',
  labelTapToCancel: 'peruuta napauttamalla',
  labelTapToRetry: 'yritä uudelleen napauttamalla',
  labelTapToUndo: 'kumoa napauttamalla',
  labelButtonRemoveItem: 'Poista',
  labelButtonAbortItemLoad: 'Keskeytä',
  labelButtonRetryItemLoad: 'Yritä uudelleen',
  labelButtonAbortItemProcessing: 'Peruuta',
  labelButtonUndoItemProcessing: 'Kumoa',
  labelButtonRetryItemProcessing: 'Yritä uudelleen',
  labelButtonProcessItem: 'Lataa',
  labelMaxFileSizeExceeded: 'Tiedosto on liian suuri',
  labelMaxFileSize: 'Tiedoston enimmäiskoko on {filesize}',
  labelMaxTotalFileSizeExceeded: 'Suurin kokonaiskoko ylitetty',
  labelMaxTotalFileSize: 'Tiedoston enimmäiskoko on {filesize}',
  labelFileTypeNotAllowed: 'Virheellisen tyyppinen tiedosto',
  fileValidateTypeLabelExpectedTypes: 'Odottaa {allButLastType} tai {lastType}',
  imageValidateSizeLabelFormatError: 'Kuvatyyppiä ei tueta',
  imageValidateSizeLabelImageSizeTooSmall: 'Kuva on liian pieni',
  imageValidateSizeLabelImageSizeTooBig: 'Kuva on liian suuri',
  imageValidateSizeLabelExpectedMinSize: 'Vähimmäiskoko on {minWidth} × {minHeight}',
  imageValidateSizeLabelExpectedMaxSize: 'Suurin koko on {maxWidth} × {maxHeight}',
  imageValidateSizeLabelImageResolutionTooLow: 'Resoluutio on liian alhainen',
  imageValidateSizeLabelImageResolutionTooHigh: 'Resoluutio on liian korkea',
  imageValidateSizeLabelExpectedMinResolution: 'Minimitarkkuus on {minResolution}',
  imageValidateSizeLabelExpectedMaxResolution: 'Suurin resoluutio on {maxResolution}',
  labelFileSizeBytes: 't',
  labelFileSizeKB: 'kt',
  labelFileSizeMB: 'Mt',
  labelFileSizeGB: 'Gt'
};

export default fi;`

**Comment by rikschennink** [comment]
> but do you mean that I need to know every translated text for every language?

No, I mean you were vague about which language and to what it translated so I couldn't determine the impact of the change.

Thanks for the Finnish locale file, I'll add it as soon as possible.

**Comment by rikschennink** [comment]
Alright, published under `4.30.0`, file validate size plugin has also been updated, added these labels:

```js
labelFileSizeBytes
labelFileSizeKilobytes
labelFileSizeMegabytes
labelFileSizeGigabytes
```
